docs/telemetry.md

```diff
@@ -1,12 +1,84 @@
+We have a Percona Privacy statement explaining how we handle your data in more detail.
+
+The telemetry feature is only included in software packages, compressed archives (tarballs), and Docker images.
+
+## Telemetry components
+
+Percona collects information using the following components:
+
+* The Percona Telemetry DB part collects the necessary metrics directly from the database and stores them in a Metrics file.
+
+* The Metrics file stores the metrics and is a standalone file located on the database host's file system.
+
+* The Telemetry Agent is an independent process running on your database host's operating system and carries out the following tasks:
+
+    * Collects OS-level metrics
+
+    * Reads the Metrics file, adds the OS-level metrics
+
+    * Sends the full set of metrics to the Percona Platform
+
+    * Collects the list of installed Percona packages using the local package manager
+
+The telemetry also uses the Percona Platform with the following components:
+
+* Telemetry Service - offers an API endpoint for sending telemetry. The service handles incoming requests. This service saves the data into Telemetry Storage.
+
+* Telemetry Storage - stores all telemetry data for the long term.
+
+## DB component
+
+The DB component extends the functionality of the database. This functionality can be implemented in the following ways, depending on the database:
+
+* Modifying the source code
+
+* Add modular components
+
+* Add self-contained extensions
+
+In Percona Server for MySQL, we have a DB component. The DB component is enabled by default during the initial database deployment. 
+
+This component does not collect the following information:
+
+* database names
+
+* user names or credentials
+
+* any user-entered value
+
+The component sends metrics gathered from the database instance to the Metrics file. This information is collected every 24 hours. Each collection creates a new Metrics file. When the file date exceeds seven days, the component removes the outdated files before creating the new file.
+
+The component creates a file in the local file system using a timestamp as the name with a `.json` extension.
+
+## Metrics file
+
+Percona stores the Metrics file in one of the following directories on the local file system. The location depends on the product.
+
+* Telemetry root path - `/usr/local/percona/telemetry`
+
+* PSMDB (mongod) root path -   `${telemetry root path}/psmdb/`
+
+* PSMDB (mongos) root path - `${telemetry root path}/psmdbs/'
+
+* PS root path -   `${telemetry root path}/ps/`
+
+* PXC root path - `${telemetry root path}/pxc/`
+
+* PG root path - `${telemetry root path}/pg/`
+
+Percona archives the telemetry history in `${telemetry root path}/history/`.
+
+### Metrics file format
+
+The Metrics file uses the Javascript Object Notation (JSON) format. Percona reserves the right to extend the current set of JSON structure attributes in the future.  
 
 An example of the data collected is the following:
 
```

Comment:
This is invalid example of metrics file. Here is a correct one:
`{
    "db_instance_id": "e83c568c-e140-11ee-8320-7e207666b18a",
    "pillar_version": "8.0.35-27",
    "active_plugins": [
        "binlog",
        "mysql_native_password",
        "sha256_password",
        "caching_sha2_password",
        "sha2_cache_cleaner",
        "daemon_keyring_proxy_plugin",
        "PERFORMANCE_SCHEMA",
        "CSV",
        "MEMORY",
        "InnoDB",
        "INNODB_TRX",
        "INNODB_CMP",
        "INNODB_CMP_RESET",
        "INNODB_CMPMEM",
        "INNODB_CMPMEM_RESET",
        "INNODB_CMP_PER_INDEX",
        "INNODB_CMP_PER_INDEX_RESET",
        "INNODB_BUFFER_PAGE",
        "INNODB_BUFFER_PAGE_LRU",
        "INNODB_BUFFER_POOL_STATS",
        "INNODB_TEMP_TABLE_INFO",
        "INNODB_METRICS",
        "INNODB_FT_DEFAULT_STOPWORD",
        "INNODB_FT_DELETED",
        "INNODB_FT_BEING_DELETED",
        "INNODB_FT_CONFIG",
        "INNODB_FT_INDEX_CACHE",
        "INNODB_FT_INDEX_TABLE",
        "INNODB_TABLES",
        "INNODB_TABLESTATS",
        "INNODB_INDEXES",
        "INNODB_TABLESPACES",
        "INNODB_COLUMNS",
        "INNODB_VIRTUAL",
        "INNODB_CACHED_INDEXES",
        "INNODB_SESSION_TEMP_TABLESPACES",
        "MyISAM",
        "MRG_MYISAM",
        "TempTable",
        "ARCHIVE",
        "BLACKHOLE",
        "ngram",
        "mysqlx_cache_cleaner",
        "mysqlx",
        "ROCKSDB",
        "rpl_semi_sync_source",
        "ROCKSDB_CFSTATS",
        "ROCKSDB_DBSTATS",
        "ROCKSDB_PERF_CONTEXT",
        "ROCKSDB_PERF_CONTEXT_GLOBAL",
        "ROCKSDB_CF_OPTIONS",
        "ROCKSDB_GLOBAL_INFO",
        "ROCKSDB_COMPACTION_HISTORY",
        "ROCKSDB_COMPACTION_STATS",
        "ROCKSDB_ACTIVE_COMPACTION_STATS",
        "ROCKSDB_DDL",
        "ROCKSDB_INDEX_FILE_MAP",
        "ROCKSDB_LOCKS",
        "ROCKSDB_TRX",
        "ROCKSDB_DEADLOCK"
    ],
    "active_components": [
        "file://component_percona_telemetry"
    ],
    "uptime": "6185",
    "databases_count": "7",
    "databases_size": "33149",
    "se_engines_in_use": [
        "InnoDB",
        "ROCKSDB"
    ],
    "replication_info": {
        "is_semisync_source": "1",
        "is_replica": "1"
    }
}`